plugins/modules/mount.py

```diff
@@ -251,46 +251,35 @@ def is_fspath_mounted(module, mount_dir, mount_over_dir):
 
     if mount_over_dir:
         # when mounting NFS, make sure we check for the NFS
-        # client mount point
-        cmd = "lsfs -l %s" % mount_over_dir
+        fs_name = mount_over_dir
     elif mount_dir:
-        cmd = "lsfs -l %s" % mount_dir
+        fs_name = mount_dir
     else:
         # should not happen
         result['msg'] = "Unexpected module FAILURE: one of the following is missing: "
         result['msg'] += ','.join(['mount_dir', 'mount_over_dir'])
         module.fail_json(**result)
 
+    cmd = "df"
     rc, stdout, stderr = module.run_command(cmd)
-
     if rc != 0:
-        return None
-
-    if rc == 0:
-        # Get the fs_name of the Filesystem
-        # check if it is listed in the first column of df command
-        ln = stdout.splitlines()[1]
-        fs_name = ln.split()[0]
-        cmd = "df"
```

Comment:
Same comment as before .. Full path name of the command.

Reply:
i will make changes to use the full path of df cmd